README.md

```diff
@@ -378,14 +378,30 @@ easier.
 You should not collect results intended for publication with development switches
 turned on.
 
-We also recommend that for 'real' benchmarking you turn on the
-`--hardware-reboots` and `--daemonise` switches. These ensure that the system
-will reboot before each benchmark execution, and that Krun will run in the
-background, allowing you to log out before the first reboot.
-
-You will also need to ensure that Krun is restarted once the machine has
-rebooted. The `etc/` directory contains example `/etc/rc.local` files for the
-platforms supported by Krun.
+We also recommend that benchmarking in "reboot mode". The recommended way to do
+this is via `cron(8)`. We supply a script `start_krun_from_cron` to make this
+easier.
+
+As a regular user (who has access to your experiment and the Krun code), invoke
+`crontab -e` and add lines similar to the following:
+
+```
+@reboot /path/to/krun/scripts/start_krun_from_cron /path/to/your/config.krun
+```
+
+The path to the config file must be an absolute path.
+
+If you wish to pass custom flags to Krun, you can append those onto the end of
+the invocation.
+
+You can then start the experiment with the same command (i.e. your crontab(5)
+line without the `@reboot`).
+
+You shouldn't log in for the duration of your experiment. Instead, you should
+add a `MAIL_TO` list into your config file and Krun will email you when your
```

Comment:
`MAIL_TO` -> `MAILTO`

Reply:
It's definitely `MAIL_TO`:
https://github.com/softdevteam/krun/blob/c96ff77ec3976f49563906e8e223c2bc5188b5fe/krun/config.py#L56

Reply:
I apologise. I thought this was cron's `MAILTO`. Maybe we could say "Add the `MAIL_TO` option in your `.krun` config file" or similar?

Reply:
Not trying to be annoying, but isn't that pretty much what it says already?

Reply:
Well, I misread it twice... Admittedly, that's not proof that the text isn't perfect, but I'm suggesting a clarification that might help idiots like me.

Reply:
I've pushed an explicit example that should make it super clear.

Reply:
Cool, thanks.